client/src/app/about/page.tsx

```diff
@@ -0,0 +1,72 @@
+import { Typography } from "@mui/material/";
+import "./styles.modules.css";
+import Link from "next/link";
+export default function Page() {
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+      }}
+    >
+      <div
+        className="border"
+        style={{
+          borderRadius: "15px",
+          padding: "25px",
+          marginBottom: "20px",
+          backgroundColor: "white",
+        }}
+      >
+        <Typography
+          className="bottom-pad"
+          variant="h3"
+          sx={{
+            fontSize: 45,
+          }}
+        >
```

Comment:
Change text to align to center. would give page a more centered look

Reply:
This was not how it was designed on the figma file.
<img width="487" alt="image" src="https://github.com/user-attachments/assets/099cfab3-6096-4c9b-ad23-0334fa719c4a">

Reply:
This looks good